# HG changeset patch
# User Max Brister <max@2bass.com>
# Date 1345092676 18000
#      Wed Aug 15 23:51:16 2012 -0500
# Node ID 8b9b3677a4d6215fff60f9b4022b92c11c8d5959
# Parent  a7a56b436de2e5d7e7106b88e26044a1e9d71829
pt-jit.h: Remove unused typedef

diff --git a/src/interp-core/pt-jit.h b/src/interp-core/pt-jit.h
--- a/src/interp-core/pt-jit.h
+++ b/src/interp-core/pt-jit.h
@@ -343,18 +343,16 @@ public:
   llvm::Module *get_module (void) const { return module; }
 
   void optimize (llvm::Function *fn);
  private:
   bool initialize (void);
 
   size_t trip_count (const octave_value& bounds) const;
 
-  // FIXME: Temorary hack to test
-  typedef std::map<tree *, jit_info *> compiled_map;
   llvm::Module *module;
   llvm::PassManager *module_pass_manager;
   llvm::FunctionPassManager *pass_manager;
   llvm::ExecutionEngine *engine;
 };
 
 class
 jit_info
